dependencies.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# $1 MUST be 'init' or 'update'.
+
+
+# get submodules
+SM_CONFIG=$(cat submodules/ocrd_all/.gitmodules)
+
+SUBMODULE_NAMES=$(
+    for LINE in $SM_CONFIG
+    do
+        echo $LINE | grep -oE '"(.+?)"' | cut -d'"' -f2
+    done
+)
+
+CURRENT_DIR=$PWD
+
+# get the main OCR-D dependencies which stem from core and that most processors have in common.
+cd 'submodules/ocrd_all/core' || exit
+python -m venv venv
+VENV=$PWD'/venv/bin/python'
+CMD='-m pip install ocrd'
+$VENV $CMD
+CMD='-m pip freeze -l'
+$VENV $CMD > $CURRENT_DIR'/core_deps.txt'
+cd $CURRENT_DIR || exit
+
+echo '{' >> $CURRENT_DIR'/deps.json'
+
+for NAME in $SUBMODULE_NAMES
+do
+    if [ $NAME != 'tesseract' ] && [ $NAME != 'opencv-python' ]; then
+        DIR='/submodules/ocrd_all/'$NAME
+        echo "Currently processing " $NAME " ..."
+        cd $CURRENT_DIR$DIR || exit
+        python -m venv venv
```

Comment:
```suggestion
        python3 -m venv venv
```

python without qualifier is python2.7 in debian